### PR TITLE
Visual tests for stochastic oscillator

### PR DESCRIPTION
**Pull request by bjedrzejewski**
I added the standard visual tests for stochastic oscillator.

### CODE REVIEW

**Comment by jdunkerley** [comment]
looks like this got mixed with the layout-css commit as well

**Comment by bjedrzejewski** [comment]
Not anymore :)

**Comment by bjedrzejewski** [comment]
@jdunkerley I think I got this whole github upstream rebase thing sorted out now :)

File: visual-tests/indicator/stochasticOscillator.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    // Create scale for y axis
+    var priceScale = d3.scale.linear()
+        .domain([0, 100]) // Percentage scale
+        .range([height, 0])
+        .nice();
+
+    // Create the stochastic oscillator component
+    var stochasticAlgo = fc.indicator.algorithm.stochasticOscillator()
+        .kWindowSize(5);
```
Comment (jdunkerley) [inline visual-tests/indicator/stochasticOscillator.js]: The default is 5 anyway, don't think we need this for the visual test but no harm.

**Comment by jdunkerley** [comment]
@bjedrzejewski getting there myself but still scared of it :)

File: visual-tests/indicator/stochasticOscillator.js (hunk 1 above)
Comment (bjedrzejewski) [inline visual-tests/indicator/stochasticOscillator.js]: Good point, I added comment to clarify.

**Comment by jdunkerley** [comment]
rest looks good to me :+1:

**Comment by chrisprice** [comment]
:+1:

**Pull request merged**